### PR TITLE
Update notification docs

### PR DESCRIPTION
**Pull request by sivchari**
**What this PR does / why we need it**:

I update docs for slack notification.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by sivchari** [comment]
@khanhtc1202 
I update docs.

File: docs/content/en/docs/user-guide/managing-piped/configuration-reference.md [hunk 1]
```diff
@@ -255,7 +255,9 @@ Must be one of the following structs:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
-| hookURL | string | The hookURL of a slack channel. | Yes |
+| hookURL | string | The hookURL of a slack channel. | No |
+| oauthToken | string | The token of a slack api. | No |
```
Comment (khanhtc1202) [inline docs/content/en/docs/user-guide/managing-piped/configuration-reference.md:259]:
```suggestion
| oauthToken | string | The token for slack api use. | No |
```
Would be best if you can also add a link to how to get slack oauth token docs.

Comment (sivchari) [reply docs/content/en/docs/user-guide/managing-piped/configuration-reference.md]: I updated it.

**Review by khanhtc1202 — DISMISSED** [review]
LGTM, thanks 🥰

**Review by khanhtc1202 — COMMENTED** [review]
Oops, sorry for my miss. Please update docs-dev instead, this feature has not yet available 🙏
docs/content/en/docs-dev/user-guide/managing-piped/configuration-reference.md

**Comment by sivchari** [comment]
@khanhtc1202 
I changed it.

File: docs/content/en/docs/user-guide/managing-piped/configuration-reference.md [hunk 2]
```diff
@@ -255,7 +255,7 @@ Must be one of the following structs:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
-| hookURL | string | The hookURL of a slack channel. | Yes |
+| hookURL | string | The hookURL of a slack channel. | No |
```
Comment (khanhtc1202) [inline docs/content/en/docs/user-guide/managing-piped/configuration-reference.md:258]: We don't need to update this, `/docs` is for the current released version of pipecd (which need the hookURL to perform send message), when we release the new version which contains your patch, the `/docs` will be synced with `/docs-dev` and show the latest docs which fit the piped functional 😄

**Review by khanhtc1202 — APPROVED** [review]

**Comment by khanhtc1202** [comment]
Thanks 🙏

**Pull request merged**